lib/vdsm/tool/configurators/multipath.py

```diff
@@ -256,6 +257,7 @@
 
 blacklist {
     protocol "(scsi:adt|scsi:sbp)"
+    devnode "^(rbd)[0-9]*"
```

Comment:
The actual change looks good, but we need @bennyz ack.

Reply:
ACK'd, this rule was suggested by ceph developers and has been confirmed by users to work

Reply:
Marking as resolved